rover-apps/science/src/AdafruitSTEMMAModule.cpp

```diff
@@ -0,0 +1,13 @@
+#include "AdafruitSTEMMAModule.h"
+#include "Module.h"
+#include "mbed.h"
+#include "AdafruitSTEMMA.h"
+
+AdafruitSTEMMAModule::AdafruitSTEMMAModule() {
+    moistureSensorModule = new AdafruitSTEMMA(TEMP_MOIST_I2C_SDA, TEMP_MOIST_I2C_SCL);
```

Comment:
You can use initializer lists to initialize the member var:
`AdafruitsSTEMMAModule::AdafruitSTEMMAModule() : moistureSensorModule(TEMP_MOIST_I2C_SDA, TEMP_MOIST_I2C_SCL) {}`